src/averageAges.js

```diff
@@ -55,7 +71,25 @@ function calculateWomenAverageAge(people, withChildren) {
  * @return {number}
  */
 function calculateAverageAgeDiff(people, onlyWithSon) {
-  // write code here
+  let average = 0;
+  const allMothers = people.map(person => person.mother);
+  const arrMothers = people.filter(person => allMothers.includes(person.name));
+  const nameMothers = arrMothers.map(person => person.name);
```

Comment:
does this variable contain only one name of some mother? or it should contain the name of mother which appears in the array several times in different objects?
Checklist # 1